src/observation/subscribeToSimpleQuery/index.js

```diff
@@ -60,28 +60,33 @@ export default function subscribeToSimpleQuery<Record: Model>(
   let unsubscribed = false
   let unsubscribe = null
 
-  query.collection
-    .fetchQuery(query)
-    .then(function observeQueryInitialEmission(initialRecords): void {
-      if (unsubscribed) {
-        return
-      }
+  query.collection._fetchQuery(query, function observeQueryInitialEmission(result): void {
+    if (unsubscribed) {
+      return
+    }
+
+    if (!result.value) {
```

Comment:
Is there a specific reason why are we checking if value is undefined instead of checking if `result.error` is defined?

Reply:
no reason. checking for `result.error` is more correct, although in this case value will never be falsy. will fix